src/components/with-datasets/redux/actions.ts

```diff
@@ -8,16 +8,9 @@ import {
 import type { SearchObject } from '../../../types';
 
 interface GetDatasetsParams {
-  uris?: string[];
+  uri?: string[];
   size?: number;
   orgPath?: string;
-  subject?: string;
-  info_model?: string;
-  referencesSource?: string;
-  accessService?: string;
-  conformsTo?: string;
-  relatedToInfoModel?: string;
-  nap?: boolean;
```

Comment:
er det ikke denne som brukes for transportportal?

Reply:
Har sjekket, ikke i bruk. @jeffreiffers har fikset for transportportal: #1964